README.md

```diff
@@ -15,6 +15,78 @@
 - VScode
 - Lots of love :heart:
 
+## Home Page
+
+![](<img width="1437" alt="Screen Shot 2021-11-21 at 2 34 10 AM" src="https://user-images.githubusercontent.com/62937819/142737425-b50692b9-5f9d-4d9a-ae64-643fba965ad4.png">
+)
+# React-Redux Bookstore 
+
+> In this project, I built a bookstore app with React and Redux.
+
+## Built with
+
+- HTML
+- CRACO
+- CSS
+- React
+- JavaScript
+- Tailwind CSS
+- VScode
+- Lots of love :heart:
+
+## Live Demo
+
+[Click here to see the project]()
+
+
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps.**
+## Setup
+
+- Clone this repository. 
+- Run ``npm i`` or ``yarn`` to install dependencies required to start the project
+- Make sure to use node v15.11.0 (npm v7.6.0) 
+- Run ``npm start``
+
+## Run Tests
+
+```
+npm run lint
+```
+
+## Author:
```

Comment:
[OPTIONAL] You have some sections repeated on your README, I advise you to look into this. Also, you can add the link to the live preview on Netlify, but if there is no live preview, better just delete that section so the visitors of the repo don't encounter 404 errors when trying to view your project.